config.js

```diff
@@ -447,5 +447,7 @@ config.tincanapi = {
  */
 config.mixpanel = {
     'enabled': false,
+    'apikey': '',
+    'apisecret': '',
     'token': 'f3e9fce119d357b745a8dfa36248d632'
```

Comment:
If we can use the token to do API requests we should, but I'm guessing it's not since you added these.